tensorflow_addons/optimizers/tests/standard_test.py

```diff
@@ -0,0 +1,49 @@
+import pytest
+import numpy as np
+import tensorflow as tf
+
+from tensorflow_addons import optimizers
+import inspect
+import sys
+
+classes_to_test = ["RectifiedAdam", "LazyAdam"]
+
+
+def discover_classes(module, parent):
+
+    classes = [
+        m[1]
+        for m in inspect.getmembers(module, inspect.isclass)
+        if issubclass(m[1], parent) and m[0] in classes_to_test
+    ]
+
+    return classes
```

Comment:
I like this, but can we make it so it discovers all optimizers within the folder and then add a list of exemptions instead of a list of things to test. IMO it is likely that newly added optimizers will not be added to the "to_test" list

Reply:
that's true, an exception list will allow us to automatically catch new opts. I'll change that.